lib/lm4f/uart.c

```diff
@@ -130,6 +130,25 @@ void uart_set_baudrate(uint32_t uart, uint32_t baud)
 	UART_FBRD(uart) = div % 64;
 }
 
+/**
+ * \brief Get UART baudrate
+ *
+ * @param[in] uart UART block register address base @ref uart_reg_base
+ * @return Baud rate in bits per second (bps)
+ */
+uint32_t uart_get_baudrate(uint32_t uart)
+{
+	/* Are we running off the internal clock or system clock? */
+	const uint32_t clock = UART_CC(uart) == UART_CC_CS_PIOSC ? 16000000 : rcc_get_system_clock_frequency();
```

Comment:
The constant for `PIOSC` mode is missing its `U` suffix, please include it so the expression is unsigned.

Reply:
Accepted. I was looking at `uart_set_baudrate()` implementation up above which sets `uint32_t clock` to 16000000 signed.